containers/server-image/Dockerfile

```diff
@@ -16,22 +16,27 @@ COPY setup_env.sh /root/setup_env.sh
 COPY uyuni-setup.service /usr/lib/systemd/system/
 
 # Main packages
+# TODO use a no duplicate remove_unused.sh
+COPY remove_unused.sh .  
+RUN echo "rpm.install.excludedocs = yes" >>/etc/zypp/zypp.conf
 # TODO Remove vi and possibly python3-simplejson
 RUN zypper ref
-RUN zypper --gpg-auto-import-keys --non-interactive install --auto-agree-with-licenses  \
+RUN zypper --gpg-auto-import-keys --non-interactive install --auto-agree-with-licenses --force-resolution \
     ${PRODUCT_PATTERN} \
     javamail \
-    libyui-ncurses-pkg16 \
-    vim \
-    python3-simplejson
+    libyui-ncurses-pkg16 
 
 
 # FIXME hack to correct the report db script to work on containers
 RUN cp /usr/bin/uyuni-setup-reportdb /usr/bin/uyuni-setup-reportdb.original
 RUN sed -i 's/sysctl kernel.shmmax/#sysctl kernel.shmmax/g' /usr/bin/uyuni-setup-reportdb
+RUN sed -i 's/-a "$PG_SOCKET"//g' /usr/bin/uyuni-setup-reportdb
 
 RUN systemctl enable uyuni-setup
 
+# Output all logs to stdout to integrate with containers tooling
+RUN sed -i 's/#ForwardToConsole=no/ForwardToConsole=yes/' /etc/systemd/journald.conf
+
```

Comment:
I dropped that one in the end as it didn't do what I expected. Looks like there is no way to output the journald data to PID1's stdout.